src/app/components/event-card/event-card.tsx

```diff
@@ -40,9 +42,7 @@ export const EventCard = ({
       </div>
       <div className="address">{address}</div>
       <div className="city">{city}</div>
-      <a href={href} target="_blank" rel="noreferrer" className="details-button">
-        View details
-      </a>
+      <LinkCustom label="View details" href={href} arrow={false} size="small" target="_blank" />
```

Comment:
I suggest to do `target="_blank"` as default parameter

Reply:
I tried to do that, but then we can't change the property when we use the component.